### DISCUSSION TITLE
Any one getting cross lingual cloning with this repo?

### DISCUSSION THREAD

**Discussion by suda-ss** [Jeevesh8/Cross-Lingual-Voice-Cloning#28] [Q&A]
Hi, I've been trying to implement this repo from quite sometime. No luck so far. I have tried this[ voice cloning](https://wavel.ai/studio/ai-voice-cloner/) tool, seems like they have figured multiligual cloning.

**Comment by mindofsteel** [comment]
> Hi, I've been trying to implement this repo from quite sometime. No luck so far. I have tried this[ voice cloning](https://wavel.ai/studio/ai-voice-cloner/) tool, seems like they have figured multiligual cloning.

Have you tried Eleven Labs's crosslingual voice cloning? It's mindblowingly good! It is paid, but it's affordable. But also, there are samples of it online, but also, let me know if you want me to create some samples for you using audio of your choice, and we can coordinate that. (I'm not affiliated or linked with Eleven Labs in any way though.)
